src/data/hooks/useVersionActions.ts

```diff
@@ -147,5 +149,36 @@ export default function useVersionActions({
     [updateTermScheduleData]
   );
 
-  return { addNewVersion, deleteVersion, renameVersion };
+  const cloneVersion = useCallback(
```

Comment:
👍 nice clean implementation.

Initially, I was slightly concerned that having two calls to `updateTermScheduleData` (one inside `addNewVersion`, and one here) might cause two separate renders if they aren't batched, but I just checked it locally and it only re-renders once in the React DevTools Profiler. No problems here 👍